tools/repeatexplorer2/.shed.yml

```diff
@@ -0,0 +1,11 @@
+---
```

Comment:
Can you convert that to a suite of tools?
In the original repo there have been two tools at least isn't it?

Reply:
I converted it to a suite.

BTW @bgruening  it looked to me like the second wrapper runs the same command as the first with an additional option `tarean_mode`, so we can just include that function in this wrapper if we want it.

Reply:
(probably)

Reply:
Yes, lets do that then. In general a suite can not hurt, so its fine to have it as suite in case we ever want to extend it. Thanks